lua/sql/defs.lua

```diff
@@ -155,416 +146,28 @@ M.flags['open_privatecache']  = 0x00040000
-M.flags['dbconfig_enable_fts3_tokenizer'] = 1004
-M.flags['dbconfig_enable_load_extension'] = 1005
-M.flags['dbconfig_no_ckpt_on_close']      = 1006
-M.flags['dbconfig_enable_qpsg']           = 1007
-M.flags['dbconfig_trigger_eqp']           = 1008
-M.flags['dbconfig_reset_database']        = 1009
-M.flags['dbconfig_defensive']             = 1010
-M.flags['dbconfig_writable_schema']       = 1011
-M.flags['dbconfig_legacy_alter_table']    = 1012
-M.flags['dbconfig_dqs_dml']               = 1013
-M.flags['dbconfig_dqs_ddl']               = 1014
-M.flags['dbconfig_enable_view']           = 1015
-M.flags['dbconfig_legacy_file_format']    = 1016
-M.flags['dbconfig_trusted_schema']        = 1017
-M.flags['dbconfig_max']                   = 1017
-
--- Authorizer Return Codes
-M.flags['deny']   = 1
-M.flags['ignore'] = 2
-
--- Authorizer Action Codes
-M.flags['create_index']        = 1
-M.flags['create_table']        = 2
-M.flags['create_temp_index']   = 3
-M.flags['create_temp_table']   = 4
-M.flags['create_temp_trigger'] = 5
-M.flags['create_temp_view']    = 6
-M.flags['create_trigger']      = 7
-M.flags['create_view']         = 8
-M.flags['delete']              = 9
-M.flags['drop_index']          = 10
-M.flags['drop_table']          = 11
-M.flags['drop_temp_index']     = 12
-M.flags['drop_temp_table']     = 13
-M.flags['drop_temp_trigger']   = 14
-M.flags['drop_temp_view']      = 15
-M.flags['drop_trigger']        = 16
-M.flags['drop_view']           = 17
-M.flags['insert']              = 18
-M.flags['pragma']              = 19
-M.flags['read']                = 20
-M.flags['select']              = 21
-M.flags['transaction']         = 22
-M.flags['update']              = 23
-M.flags['attach']              = 24
-M.flags['detach']              = 25
-M.flags['alter_table']         = 26
-M.flags['reindex']             = 27
-M.flags['analyze']             = 28
-M.flags['create_vtable']       = 29
-M.flags['drop_vtable']         = 30
-M.flags['function']            = 31
-M.flags['savepoint']           = 32
-M.flags['copy']                = 0
-M.flags['recursive']           = 33
-
--- SQL Trace Event Codes
-M.flags['trace_stmt']    = 0x01
-M.flags['trace_profile'] = 0x02
-M.flags['trace_row']     = 0x04
-M.flags['trace_close']   = 0x08
-
--- Run-Time Limit Categories
-M.flags['limit_length']              = 0
-M.flags['limit_sql_length']          = 1
-M.flags['limit_column']              = 2
-M.flags['limit_expr_depth']          = 3
-M.flags['limit_compound_select']     = 4
-M.flags['limit_vdbe_op']             = 5
-M.flags['limit_function_arg']        = 6
-M.flags['limit_attached']            = 7
-M.flags['limit_like_pattern_length'] = 8
-M.flags['limit_variable_number']     = 9
-M.flags['limit_trigger_depth']       = 10
-M.flags['limit_worker_threads']      = 11
-
--- Prepare Flags
-M.flags['prepare_persistent'] = 0x01
-M.flags['prepare_normalize']  = 0x02
-M.flags['prepare_no_vtab']    = 0x04
-
--- Fundamental Datatypes
```

Comment:
Hmm I remember coping those to sqlite_datatypes, I guess they will not be needed outside stmt.lua

Reply:
Oh right. Do we need to keep those flags seem useful

Reply:
I vote keep them, then later I'll remove sqlite_datatype

Reply:
Added back